extract_unique_sensors_01.py

```diff
@@ -95,18 +110,25 @@ def splitting_extracted_sensors(candidate_sensors, sensor_name):
     sensors_list = []
     for sensor in sorted(candidate_sensors):
         # Typical output from "sensor" variable:
-        # kat.sensors.anc_api_version   kat.sensors.anc_ganglia_api_version   kat.sensors.anc_ganglia_kat_monctl_system_os_name
-        sensor_ = sensor.split('.')[-1]  
-        sensor_ = sensor_.strip("\n") 
+        # kat.sensors.anc_api_version   kat.sensors.anc_ganglia_api_version
+        # kat.sensors.anc_ganglia_kat_monctl_system_os_name
```

Comment:
Perhaps install flake8 plugin that detects any dead code in your codebase...

Run: `https://pypi.org/project/flake8-eradicate/` and re-run `flake8` on your file.

Reply:
Done!

Reply:
Didn't you get any errors?

Reply:
This plugin should have reported either warnings or errors relating to line 117 and 118. Commented code should never be committed unless there's a detailed comment.
But it's fine since there's that message in line 116